src/ApplicationLogs/LogReader.cs

```diff
@@ -186,6 +197,33 @@ private void OnCommitting(NeoSystem system, Block block, DataCache snapshot, IRe
             }
         }
 
+        private static void ApplicationEngine_Log(object sender, LogEventArgs args)
+        {
+            if (!Settings.Default.Debug) return;
+
+            var tx = ((Transaction)args.ScriptContainer).Hash;
```

Comment:
It can be not only a transaction, right? E.g. block for OnPersist and PostPersist execution results.

Reply:
> It can be not only a transaction, right? E.g. block for OnPersist and PostPersist execution results.

I think it can only be transaction, but i might be wrong. May you please confirm @shargon @roman-khimov